imports/client/components/ProfileList.tsx

```diff
@@ -367,6 +371,67 @@ const ProfileList = ({
     );
   }, [hunt, canSyncDiscord, syncDiscord]);
 
+  const invitationLink = useMemo(() => {
+    if (!hunt || !canInvite || !hunt.invitationCode) {
+      return null;
+    }
+
+    return (
+      <p>
+        Invitation link:{" "}
+        <a href={`/join/${hunt.invitationCode}`}>
+          {`${window.location.origin}/join/${hunt.invitationCode}`}
+        </a>
```

Comment:
Should this be a copy-to-clipboard button instead of a link?

Reply:
Changed to plain text with a copy-to-clipboard button (cribbed from the guess queue page).